### PR TITLE
Enable computed goto for OSX

### PR DESCRIPTION
**Pull request by charliegracie**
Signed-off-by: Charlie Gracie <charlie.gracie@gmail.com>

### CODE REVIEW

**Comment by charliegracie** [comment]
Jenkins compile osx jdk11

**Comment by charliegracie** [comment]
Jenkins compile osx jdk11

**Comment by charliegracie** [comment]
Jenkins compile osx jdk11

**Comment by charliegracie** [comment]
Jenkins compile osx jdk11

**Comment by AdamBrousseau** [comment]
Jenkins copyright check

**Comment by AdamBrousseau** [comment]
Jenkins line endings check

**Comment by jdekonin** [comment]
Jenkins compile osx jdk11

**Comment by charliegracie** [comment]
@gacholio I think this is ready now

File: runtime/vm/BytecodeInterpreter.cpp [hunk 1]
```diff
@@ -36,7 +36,11 @@
 
 /* USE_COMPUTED_GOTO on Windows is a performance improvement of 15% */
 /* USE_COMPUTED_GOTO on Linux amd64 has a performance improvement of 3% */
-#if (defined(WIN32) && defined(__GNUC__)) || (defined(LINUX) && defined(J9HAMMER))
+#if (defined(WIN32) && defined(__GNUC__))
+#define USE_COMPUTED_GOTO
+#elif (defined(LINUX) && defined(J9HAMMER))
+#define USE_COMPUTED_GOTO
+#elif defined(OSX)
```
Comment (gacholio) [inline runtime/vm/BytecodeInterpreter.cpp]: Why not just tack this on the end of the existing line?

Comment (charliegracie) [reply runtime/vm/BytecodeInterpreter.cpp]: I felt like the line was getting long and difficult to read. I can switch it back if you prefer

**Pull request merged**